server/src/main/java/io/druid/segment/realtime/plumber/Sink.java

```diff
@@ -262,7 +280,7 @@ private FireHydrant makeNewCurrIndex(long minTimestamp, DataSchema schema)
     final IncrementalIndex newIndex = new IncrementalIndex.Builder()
         .setIndexSchema(indexSchema)
         .setReportParseExceptions(reportParseExceptions)
-        .setMaxRowCount(maxRowsInMemory)
+        .setMaxRowCount(maxRowsInMemory).setMaxBytesInMemory(maxBytesInMemory)
```

Comment:
Please break the line before `.setMaxBytesInMemory()`.

Reply:
done